scripts/known_good.json

```diff
@@ -52,11 +52,11 @@
     },
     {
       "name" : "Vulkan-ValidationLayers",
-      "url" : "https://github.com/KhronosGroup/Vulkan-ValidationLayers.git",
+      "url" : "c:/dev/xgl/Vulkan-ValidationLayers.git",
```

Comment:
wha?

Reply:
Ya, sorry, had some leftover cruft in there.  Cleaned up and force-pushed.